api/api.go

```diff
@@ -18,8 +18,10 @@ func Serve(router *mux.Router, store *storage.Storage, pool *worker.Pool, feedHa
 
 	sr := router.PathPrefix("/v1").Subrouter()
 	middleware := newMiddleware(store)
+	sr.Use(middleware.handleCORS)
 	sr.Use(middleware.apiKeyAuth)
 	sr.Use(middleware.basicAuth)
+	sr.Methods("OPTIONS")
```

Comment:
Why `OPTIONS` needs to be here?

Reply:
To handle the [preflight requests](https://developer.mozilla.org/en-US/docs/Glossary/Preflight_request).